device/core/src/client.ts

```diff
@@ -443,10 +458,13 @@ export class Client extends EventEmitter {
    *
    */
   getTwin(done: (err?: Error, twin?: Twin) => void, twin?: Twin): void {
```

Comment:
> twin?: Twi [](start = 51, length = 11)

Still needed?

Reply:
nope! good catch!

---
In reply to: [171919575](https://github.com/pierreca/azure-iot-sdk-node/pull/2#discussion_r171919575) [](ancestors = 171919575)